tests/cli/test_cli_core.py

```diff
@@ -1,6 +1,6 @@
 import subprocess
 from datetime import datetime
-from typing import Dict, Optional
+from typing import Optional
```

Comment:
Also in this file some type hints that could be updated to python 3.11 stardards